src/System.Reflection.Context/src/System/Reflection/Context/Delegation/DelegatingAssembly.cs

```diff
@@ -98,8 +98,7 @@ public override Type[] GetExportedTypes()
         {
             return UnderlyingAssembly.GetExportedTypes();
         }
-
-        [SecurityCritical]
+        
```

Comment:
It seems that the removal left all the spaces at the beginning of the line, so now it's a line of non-zero whitespace.

Reply:
you're right

Reply:
@bartonjs is there a way to see that on "web client"?

Reply:
Highlight the diff region, the whitespace characters get an inverted background, but the "there's just screen here" doesn't.

Reply:
it would be useful "View White Space" VS style.